sdk/azcore/version.go

```diff
@@ -10,5 +10,5 @@ const (
 	UserAgent = "azcore/" + Version
 
 	// Version is the semantic version (see http://semver.org) of the pipeline package.
-	Version = "0.13.0"
+	Version = "0.13.3"
```

Comment:
Since `v0.13.3` has already been tagged this won't get picked up.

I've been thinking about this problem in general.  At the very minimum, for PRs into `master` we should have a check to ensure that `version.go` is being updated and if it isn't to fail CI.

Reply:
The check sounds like a good idea to me. I'll remove this change

Reply:
In addition, we don't even inject `UserAgent` into the telemetry string :(  I will take a look at fixing this (it's missing for `armcore`, I haven't checked `azidentity`).

Reply:
`azidentity` doesnt do it currently, if I recall correctly.